CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Unreleased
```

Comment:
@getsentry/ingest is there anything wrong with my changelog entry here?

Reply:
Yes, see below for my suggestion.